build.sbt

```diff
@@ -54,13 +55,16 @@ lazy val awslambda = alpakkaProject("awslambda",
                                     // For mockito https://github.com/akka/alpakka/issues/390
                                     parallelExecution in Test := false)
 
-lazy val azureStorageQueue = alpakkaProject("azure-storage-queue", Dependencies.AzureStorageQueue)
+lazy val azureStorageQueue =
+  alpakkaProject("azure-storage-queue", Dependencies.AzureStorageQueue)
```

Comment:
Could you reformat this file as it was (except for the orientDB line obviously)?
There is no need to re-format the code if no changes are made